ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+
+puts "How many employees would you like to proess?"
+num_of_employees = gets.chomp
+count = num_of_employees.to_i
+
+while count > 0
+	puts "What is your name?"
+	employee_name = gets.chomp
+
+	puts "How old are you? What year were you born?"
+	employee_age = gets.chomp
+	employee_birth_year = gets.chomp
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+	wants_garlic = gets.chomp
+
+	puts "Would you like to enroll in the company’s health insurance? (y/n)"
+	wants_insurance = gets.chomp
+
+	got_age_right = true
+	likes_garlic = true
+	agreed_to_insurance = true
+
+	if 2017-employee_age.to_i != employee_birth_year.to_i
```

Comment:
Check out the Ruby Time class and see what you can do with `Time.now.year`.  This will help the code run a but more dynamically.  Otherwise you will have to keep changing this file every new year to keep the code running properly.